compute/cloud-client/src/main/java/compute/DeleteInstance.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package compute;
+
+// [START compute_instances_delete]
+
+import com.google.cloud.compute.v1.InstancesClient;
+import com.google.cloud.compute.v1.Operation;
+import com.google.cloud.compute.v1.Operation.Status;
+import com.google.cloud.compute.v1.ZoneOperationsClient;
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+
+public class DeleteInstance {
+
+  public static void main(String[] args) throws IOException, InterruptedException {
+    // TODO(developer): Replace these variables before running the sample.
+    String project = "your-project-id";
+    String zone = "zone-name";
+    String machineName = "machine-name";
+    deleteInstance(project, zone, machineName);
+  }
+
+  // deletes the instance as specified by the machineName
+  // if it is present in the given project and zone
+  public static void deleteInstance(String project, String zone, String machineName)
+      throws IOException, InterruptedException {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources.
+    try (InstancesClient instancesClient = InstancesClient.create()) {
+
+      System.out.println(String.format("Deleting instance: %s ", machineName));
+      // set the properties of the instance which is to be deleted
+      Operation response = instancesClient.delete(project, zone, machineName);
+
+      // [START compute_instances_operation_check]
```

Comment:
This snippet won't have the import statements, wrapper, or anything. Does this region tag really need to be used here, or should it be moved into it's own snippet?

Reply:
There's been an issue with moving operation check to its own snippet. Hence, we decided to follow the same practice across samples in all languages. 
Please refer to php sample here: https://github.com/GoogleCloudPlatform/php-docs-samples/blob/master/compute/cloud-client/instances/src/delete_instance.php